src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -88,6 +90,19 @@ public class MissingSwitchDefaultCheck extends AbstractCheck {
      */
     public static final String MSG_KEY = "missing.switch.default";
 
+    /**
+     * Parents of switch statement.
+     */
```

Comment:
`Parents of switch statement`. => `Represents the possible parent tokens of a switch statement`

Reply:
Done